### PR TITLE
Remove dependency on ember-composable-helpers

### PR DESCRIPTION
**Pull request by stopfstedt**
fixes https://github.com/ilios/ilios/issues/4491
refs https://github.com/ilios/ilios/issues/4492

### CODE REVIEW

File: app/helpers/intersection-count.js [hunk 1]
```diff
@@ -1,42 +0,0 @@
-/* eslint ember/avoid-leaking-state-in-ember-objects: 0 */
```
Comment (jrjohnson) [inline app/helpers/intersection-count.js:1]: Ooh, happy to see this go. Were we just not using it anywhere?

**Review by jrjohnson — APPROVED** [review]

File: app/helpers/intersection-count.js (hunk 1 above)
Comment (stopfstedt) [reply app/helpers/intersection-count.js:1]: yea, was surprised too. we must have left this behind some while ago.

**Review by dartajax — CHANGES_REQUESTED** [review]
image

**Comment by dartajax** [comment]
It looks identical if only for the Browser message behind.

**Review by dartajax — APPROVED** [review]
I found one weird browser message that could not be duplicated a second time. I click tested through the application - I find these changes to be fine. Planning on merging this soon.

**Pull request merged**